tests/e2e/etcd_grpcproxy_test.go

```diff
@@ -59,7 +57,7 @@ func TestGrpcProxyAutoSync(t *testing.T) {
 		assert.NoError(t, proxyProc.Stop())
 	}()
 
-	proxyCtl, err := e2e.NewEtcdctl(&e2e.EtcdProcessClusterConfig{}, []string{proxyClientURL})
```

Comment:
Among all fields of `EtcdProcessClusterConfig`, only `ClientTLS`, `IsClientAutoTLS` and `IsClientCRL` are visited in the life cycle of `proxyCtl`, and these 3 fields are not modified by `e2e.DefaultConfig()`. It should be safe to replace `e2e.EtcdProcessClusterConfig{}` with `e2e.DefaultConfig()`,